notsobot/notsobot.py

```diff
@@ -1592,12 +1682,16 @@ async def invert(self, ctx, urls: ImageFinder = None):
 
             def invert_img(b):
                 with Image.open(b).convert("RGB") as img:
-                    img = ImageOps.invert(img)
+                    image = ImageOps.invert(img)
+                    img.close()
```

Comment:
This is unnecessary since `img` was created by the context manager which closes it automatically.